topics/introduction/tutorials/galaxy-intro-short/tutorial.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: tutorial_hands_on
+topic_name: introduction
+tutorial_name: galaxy-intro-short
+---
+
+# Overview
+{:.no_toc}
+
+* This is a short introduction to the Galaxy user interface - the web page that you interact with.
+* We will cover key tasks in Galaxy: uploading files, using tools, and viewing histories.
+
+> ### Agenda
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## What does Galaxy look like?
+
+* Browse to your Galaxy instance and log in or register.
+    * Use Chrome, Safari or Firefox as your browser, not Internet Explorer.
+    * This is an image of Galaxy Australia, located at [www.usegalaxy.org.au](https://usegalaxy.org.au/).
+
+* The particular Galaxy server that you are using may look slightly different and have a different web address.
+    * For example, the main Galaxy server is [www.usegalaxy.org](https://usegalaxy.org/).
+
+![login](../../images/galaxy-login.png)
+
+
+* The Galaxy page is divided into three panels:
+    * Tools on the left
+    * Viewing panel in the middle
+    * History of analysis and files on the right
+
+![galaxy overview screenshot](../../images/galaxy-panel.png)
+
+
+* The first time you use Galaxy, there will be no files in your history panel.
+
+
+# Key Galaxy actions
+
+## Name your current history
+
+Your "History" is in the panel at the right.
+
+> ### {% icon hands_on %} Hands-on: Name history
+> * Go to the History panel
+> * Click on the history name ("Unnamed history")
+> * Type in a new name, for example, "My-Analysis"
+> * Press Enter
+> <br>
+>
+> <img src="../../images/name_history.png" style="width: 320px;"/>
+{: .hands_on}
+
+## Upload a file
+
+Your "Tools" are in the panel at the left.
+
+> ### {% icon hands_on %} Hands-on: Upload a file
+> * Go to the Tools panel
+> * Click "Get Data" (at the top of the list)
+> * Click "Upload File"
+>
+> This brings up a box:
+> <br>
+>
+> <img src="../../images/upload-box.png" alt="filebox" style="width: 620px;"/>
+> <br>
+> * Click "Paste/Fetch data"
+> * Paste in the address of a file: *https://zenodo.org/record/582600/files/mutant_R1.fastq*
+> * Then click "Start".
+> * Then click "Close".
+> Your uploaded file is now in your current history.
+{: .hands_on}
+
+
+When the file has uploaded to Galaxy, it will turn green.
```

Comment:
Maybe you could re-use here the same comment box as in other tutorial: 

```
> ### {% icon comment %} Comment
> After this you will see your first history item in Galaxy's right panel. It will go through
> the gray (preparing/queued) and yellow (running) states to become green (success):
>
{: .comment}
```

What do you think?